inc/functions_upload.php

```diff
@@ -313,6 +313,9 @@ function upload_avatar($avatar=array(), $uid=0)
 		case "image/x-png":
 			$img_type = 3;
 			break;
+		case "image/bmp":
```

Comment:
According to Google image/bmp is an official mime type

For other MIME types, I will add them - at least image/x-windows-bmp